pubsub/apiserver/messages.go

```diff
@@ -36,6 +36,18 @@ type Details struct {
 	LocalOnly bool                 `yaml:"local-only"`
 }
 
+// DetailsRequestTopic is the topic that details requests are
+// published on. In response to those requests, the current details
```

Comment:
Perhaps mention that it is the peer grouper that replies, and that it is local only because the details topic is local only and we only want our peer grouper to get this message.

Reply:
Will do.